integrations/github-pr-review-ai-assistant/agent.py

```diff
@@ -0,0 +1,114 @@
+import os
+
+from ai_engine import UAgentResponse, UAgentResponseType
+from codedog.actors.reporters.pull_request import PullRequestReporter
+from codedog.chains import CodeReviewChain, PRSummaryChain
+from codedog.retrievers import GithubRetriever
+from github import Github
+from github.GithubException import UnknownObjectException
+from langchain.chat_models import ChatOpenAI
+from uagents import Agent, Context, Model, Protocol
+from uagents.setup import fund_agent_if_low
+
+OPENAI_API_KEY = os.getenv("OPENAI_API_KEY")
+AGENT_MAILBOX_KEY = os.getenv("AGENT_MAILBOX_KEY")
+SEED_PHRASE = os.getenv("SEED_PHRASE")
+
+agent = Agent(
+    name="Github PR review AI Assistant",
+    seed=SEED_PHRASE,
+    mailbox=f"{AGENT_MAILBOX_KEY}@https://agentverse.ai",
+)
+
+fund_agent_if_low(agent.wallet.address())
+
+
+class CodeReviewRequest(Model):
+    """Model for handling code review requests."""
+
+    repository: str
+    pull_request_number: int
```

Comment:
I would reduce this to single `pr_url` parameter
```suggestion
    pr_url: str
```